tests/fsharpqa/Source/Conformance/InferenceProcedures/NameResolution/RequireQualifiedAccess/OnRecordVsUnion.fs

```diff
@@ -0,0 +1,11 @@
+// #Conformance #TypeInference #Attributes 
+// Verify the RequireQualifiedAccess attribute works on unions
+
+module A =
+    [<RequireQualifiedAccess>]
+    type U = | C
+
+    type C() =
+        static member M() = ()
+
+let x = A.C.M()
```

Comment:
as already described. this breaking and https://github.com/Microsoft/visualfsharp/pull/1814 show that this case builds in current master. It will break after this PR.